README.md

```diff
@@ -1 +1,155 @@
+- [📝 License](#license)
+
+# 📖 Capstone 2: TV Series <a name="about-project"></a>
+
+**Capstone 2: TV Series**
+
+Video Intro: [Video Introduction](https://www.loom.com/share/831fc44d96704fc6a05630c4ec0ce66c)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>Git</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **HTML**
+- **CSS**
+- **JavaScript**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo!](https://fdezcaminero.github.io/capstone2/dist/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+* To clone the repository
+* A browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd desired-folder
+  git clone https://github.com/fdezcaminero/aGameOfGo.git
+```
+
+### Install
+
+Install this project with:
+
+* A functioning browser
+
+### Usage
+
+To run the project, execute the following command:
+
+* Open the index.html file with a functioning browser
+
+### Run tests
+
+To run tests, run the following command:
+
+* Open your dev tools while running the project on your browser
+
+### Deployment
+
+You can deploy this project using:
+
+* We are not currently allowing this project to be deployed by fans.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Authors
+
+
+👤 **Ulises Cappa**
+
+- GitHub: [@ulises2607]([https://github.com/fdezcaminero](https://github.com/ulises2607))
+- LinkedIn: [Ulises Cappa](https://www.linkedin.com/in/cesar-ulises-cappa-subelza/)
+
+
+👤 **José Fernández-Caminero**
+
+- GitHub: [@fdezcaminero](https://github.com/fdezcaminero)
+- LinkedIn: [José Fernández-Caminero](https://www.linkedin.com/in/fdezcaminero/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Responsiveness tailor made for tablets]**
+- [ ] **[Sign up/log in functionality]**
+- [ ] **[User commenting features]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project send thanks to Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] the link to the MIT license is not working as expected please make sure it redirects to the actual license.